### ISSUE TITLE
[error] Example code does not work

### ISSUE THREAD

**Issue by ssantos21** [bitcoindevkit/rust-electrum-client#144]
If you run the example code, you' ll get the error:

``` rust
let client = Client::new("ssl://electrum.blockstream.info:50002").unwrap();
let res = client.server_features();
println!("{:#?}", res);

```

```
Err(
    AllAttemptsErrored(
        [
            JSON(
                Error("EOF while parsing a value", line: 1, column: 0),
            ),
        ],
    ),
)
```

I tried other Electrum servers and the results were the same.
In my Cargo.toml the version is 0.21.0.
```toml
[dependencies]
electrum-client = "0.21.0"
```

**Comment by sansosti** [comment]
same error here

**Comment by oleonardolima** [comment]
It's weird, it worked once for me and then it started failing.

**Comment by sansosti** [comment]
I run the example code again today and it worked fine.
Something going on with electrum.blockstream.info:50002?

**Comment by oleonardolima** [comment]
> I run the example code again today and it worked fine. Something going on with electrum.blockstream.info:50002?

That would be my guess, but still, I think it can be a better error message if it's a specific one from their side.

**Comment by ssantos21** [comment]
I don't think so. If you try other servers (Electrum wallet shows multiple servers), you will notice the same issue.
Maybe something intermittent?

**Comment by sansosti** [comment]
definitely something intermittent, as - right after I said it was working fine for me - it started failing again!
